client/js/utils.js

```diff
@@ -37,6 +39,15 @@ function resetHeight(element) {
 	element.style.height = element.style.minHeight;
 }
 
+// Given a channel element will determine if the lounge user is Op in that channel
+function isOpInChannel(channel) {
+	const channelID = channel.data("id");
+	const network = $("#sidebar .network").has(`.chan[data-id="${channelID}"]`);
+	const ownNick = network.data("nick");
+	const isOP = channel.find(`.users .user-mode.op .user[data-name="${escape(ownNick)}"]`).length;
```

Comment:
@xPaw, isn't there a better way to determine if the current user is an op? If I read this correctly, this reads the info from the HTML of the user list.

Reply:
this was discussed on IRC, that's basically the only way at the moment.

Reply:
Gotcha. If there is a better way that could be possible when refactoring this or that, do we want to add a TODO note here?